.github/workflows/_e2e.yaml

```diff
@@ -23,6 +23,7 @@ jobs:
   e2e:
     timeout-minutes: 60
     runs-on: ${{ fromJSON(inputs.runnersByArch)[fromJSON(inputs.matrix).arch] }}
+    if: ${{ fromJSON(inputs.runnersByArch)[fromJSON(inputs.matrix).arch] != '' }}
```

Comment:
It will not be needed after the latest master is merged, otherwise it was needed because I tried to enable `ci/run-full-matrix` and it failed on ARM64 tasks without this because no runners available.